api/v1beta1/disruption_types_test.go

```diff
@@ -7,13 +7,17 @@ package v1beta1_test
 
 import (
 	"sort"
+	"time"
 
 	. "github.com/DataDog/chaos-controller/api/v1beta1"
+	builderstest "github.com/DataDog/chaos-controller/builderstest"
```

Comment:
```suggestion
	"github.com/DataDog/chaos-controller/builderstest"
```

no-op alias

Reply:
I don't really know why, but my IDE _(GoLand)_ forced me to use an alias. Maybe, because the package is dedicated for testing purpose (_test suffix).